src/renderer/components/ft-share-button/ft-share-button.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+	<ft-icon-button
+		title="Share Video"
+		theme="secondary"
+		icon="share-alt"
```

Comment:
The dropdown gets cut off on certain screen sizes.  We can fix this by adding `dropdown-position-x="left"` here to align it differently.